src/fsharp/PrettyNaming.fs

```diff
@@ -131,10 +131,20 @@ module internal Microsoft.FSharp.Compiler.PrettyNaming
         let nameLen = name.Length
         let rec loop i = (i < nameLen && (opCharSet.Contains(name.[i]) || loop (i+1)))
         loop 0
+    
+    /// Returns the plain name of a given identifier, operator or double backticked name, 
+    /// e.g. "|>>" instead of "( |>> )" or "long identifier" instead of "( long identifier )".
+    let DemangleBacktickedName (name:string) =
```

Comment:
The name of the function seems odd - This seems to deal with parenthesized names, not backtick names?